src/components/EmojiPicker/EmojiPicker.tsx

```diff
@@ -190,6 +192,6 @@ const EmojiPicker = forwardRef((props, ref) => {
     );
 });
 
-EmojiPicker.propTypes = propTypes;
 EmojiPicker.displayName = 'EmojiPicker';
```

Comment:
`Property 'displayName' does not exist on type '(props: EmojiPickerProps & RefAttributes<unknown>) => ReactElement<any, string | JSXElementConstructor<any>> | null'`
I am facing this issue over here, how can this be dealt?

Reply:
Hi @abdulrahuman5196 please look at this as this is causing some issue in the component. Do you have an idea?

Reply:
I think its due to usage of forwardRef here https://github.com/Expensify/App/pull/34162/files#diff-eba20b923a2357dff4d39f91a106832801db71ad38932473581c84db7cc52e7dR35

Can you try changing the  use of forwardRef in the export line?

Reply:
@lancer-spaid